CHANGELOG.md

```diff
@@ -65,6 +65,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Add a single node docker based development environment [#311](https://github.com/provenance-io/provenance/issues/311)
   * Add make targets `devnet-start` and `devnet-stop`
   * Add `networks/dev/mnemonics` for adding accounts to development environment
+* When deleting a scope, require the same permissions as when updating it [#473](https://github.com/provenance-io/provenance/issues/473)
```

Comment:
Do you want to move these up to the unreleased section instead of in v1.7.0?

Reply:
Good catch! Fixed.